include/nbl/video/IGPUDescriptorSet.h

```diff
@@ -28,10 +30,131 @@ class NBL_API IGPUDescriptorSet : public asset::IDescriptorSet<const IGPUDescrip
 		using base_t = asset::IDescriptorSet<const IGPUDescriptorSetLayout>;
 
 	public:
```

Comment:
because of the `reset` https://github.com/Devsh-Graphics-Programming/Nabla/pull/345/files#r1046890916 we need a public `bool isZombie() const` method

Reply:
Resolved.